### PR TITLE
JavaScript Formplayer Tests

### PR DESCRIPTION
**Pull request by wpride**
buddy @benrudolph should cover the core menu routes

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_view.js [hunk 1]
```diff
@@ -295,7 +295,7 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
         template: "#case-tile-grid-view-item-template",
     });
 
-    MenuList.GridCaseTileListView = MenuList.CaseTileListView.extend({
+        MenuList.GridCaseTileListView = MenuList.CaseTileListView.extend({
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_view.js]: Expected indentation of 4 space characters but found 8. (indent)

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/util/util.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/*global Backbone, FormplayerFrontend */
+/*global Backbone, FormplayerFrontend, MenuList */
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/util/util.js]: 'MenuList' is defined but never used (no-unused-vars)

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/fixtures.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+hqDefine("corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/fixtures.js", function() {
+   return {
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/fixtures.js]: Expected indentation of 4 space characters but found 3. (indent)

**Comment by benrudolph** [comment]
hot 🐶 ! dig it. robust formplayer here we come

**Comment by benrudolph** [comment]
```
Running "mocha:cloudcare" (mocha) task
Testing: http://localhost:8000/mocha/cloudcare/
Warning: TypeError: 'null' is not an object (evaluating 'document.body.style')
    at undefined (http://localhost:8000/static/css-grid-polyfill-binaries/css-polyfills.js:6519)
    at undefined (http://localhost:8000/static/css-grid-polyfill-binaries/css-polyfills.js:6671)
    at undefined (http://localhost:8000/static/css-grid-polyfill-binaries/css-polyfills.js:6683) Use --force to continue.
Aborted due to warnings.
```

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_view.js [hunk 4]
```diff
@@ -278,8 +278,10 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
         childView: MenuList.CaseTileView,
         initialize: function (options) {
             MenuList.CaseTileListView.__super__.initialize.apply(this, arguments);
+            var gridPolyfillPath = FormplayerFrontend.request('gridPolyfillPath');
             generateCaseTileStyles(options.tiles);
             makeInnerGridStyle(options.maxHeight, options.maxWidth, options.numEntitiesPerRow);
+            $.getScript(gridPolyfillPath);
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_view.js]: this is technically an async call. so we'll probably run into a race condition on prod. locally the script will load almost instantaneously but it's not going to do that on prod. this returns a promise so it's good to execute this and then use `.done` to ensure it's loaded before doing any logic

Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_view.js]: (offlined, this is fine as is)

**Pull request merged**